components/ILIAS/UICore/classes/class.ilGlobalPageTemplate.php

```diff
@@ -105,7 +107,8 @@ protected function prepareBasicJS(): void
         $sessionReminder = new ilSessionReminderGUI(
             ilSessionReminder::byLoggedInUser(),
             $this,
-            $this->lng
+            $this->lng,
+            $this->logger_factory
```

Comment:
@thibsy I had to change this file to retrieve the logger factory from the DIC and inject it to the session reminder instance.